Util/scripts/filter_fastq.py

```diff
@@ -0,0 +1,54 @@
+from typing import Dict, Tuple
+import os
+
+
+def calculate_gc_content(sequence: str) -> float:
+    """Calculates the GC content percentage in the sequence."""
+    gc_count = sequence.upper().count("G") + sequence.upper().count("C")
+    return (gc_count / len(sequence)) * 100
```

Comment:
Здесь было бы неплохо добавить проверку, чтобы избежать деления на ноль, если последовательность пустая. Типа такого
```suggestion
def calculate_gc_content(sequence: str) -> float:
    """Calculates the GC content percentage in the sequence."""
    gc_count = sequence.upper().count("G") + sequence.upper().count("C")
    if len(sequence) == 0:
        return 0.0
    return (gc_count / len(sequence)) * 100
```